packages/terra-layout/src/_IconPin.jsx

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable */
```

Comment:
Is this a new icon we'll be getting in the icon repo UX has?

Reply:
Correct, we can use that once released